executor/load_data.go

```diff
@@ -48,18 +50,62 @@ import (
 )
 
 const (
+	// LoadDataFormatCSV represents the data source file is of CSV or TSV format
+	LoadDataFormatCSV = "csv"
```

Comment:
I found a better name "delimited text file"

Reply:
add a task in task list, CSV and delimited data have different default settings like `field terminator`, will change it after pm confirms

Reply:
Current spec does not allow `LOAD DATA ... FORMAT 'csv'`

Reply:
a little anti-pattern we cannot set default format explicitly

we acutally uses `load data... format ''`, but sadly it panics now. we can fix it in another pr

Reply:
They are two different statements, FORMAT statement can not specify FIELDS/LINES options, it's protected in parser.y

Reply:
`load data local infile '/small.csv' format '' into table t;` try this, not protected enough

Reply:
I'll fix it later